CHANGELOG.md

```diff
@@ -1,3 +1,78 @@
+### citus v10.2.0 (September 11, 2021) ###
+
+* Adds PostgreSQL 14 support
+
+* Adds hash & btree index support for columnar tables
+
+* Adds helper UDFs for easy time partition management:
+  `get_missing_time_partition_ranges`, `create_time_partitions`, and
+  `drop_old_time_partitions`
+
+* Adds propagation of ALTER SEQUENCE
+
+* Adds support for ALTER INDEX ATTACH PARTITION
+
+* Adds support for CREATE INDEX ON ONLY
+
+* Allows more graceful failovers when replication factor > 1
+
+* Enables chunk group filtering to work with Params for columnar tables
+
+* Enables qual push down for joins including columnar tables
+
+* Enables transferring of data using binary encoding by default on PG14
+
+* Improves `master_update_table_statistics` and provides distributed deadlock
+  detection
+
+* Includes `data_type` and `cache` in sequence definition on worker
+
+* Makes sure that table exists before updating table statistics
```

Comment:
I think the only PR missing is https://github.com/citusdata/citus/pull/5144

I just added description there, can you move here as well?